packages/sync-actions/test/customer-group-sync.spec.js

```diff
@@ -1,38 +1,136 @@
 import customerGroupSyncFn, { actionGroups } from '../src/customer-group'
 import { baseActionsList } from '../src/customer-group-actions'
 
-describe('Exports', () => {
+describe('Discount Groups Exports', () => {
   it('action group list', () => {
-    expect(actionGroups).toEqual(['base'])
+    expect(actionGroups).toEqual(['base', 'custom'])
   })
 
-  it('correctly defined based actions list', () => {
-    expect(baseActionsList).toEqual([
-      { action: 'changeName', key: 'name' },
-      { action: 'setKey', key: 'key' },
-    ])
+  describe('action list', () => {
+    it('should contain `changeName` action', () => {
+      expect(baseActionsList).toEqual(
+        expect.arrayContaining([{ action: 'changeName', key: 'name' }])
+      )
+    })
+
+    it('should contain `setKey` action', () => {
+      expect(baseActionsList).toEqual(
+        expect.arrayContaining([
+          {
+            action: 'setKey',
+            key: 'key',
+          },
+        ])
+      )
+    })
   })
 })
 
-describe('Actions', () => {
+describe('Discount Groups Actions', () => {
   let customerGroupSync
   beforeEach(() => {
     customerGroupSync = customerGroupSyncFn()
   })
 
-  it('should build `changeName` action', () => {
-    const before = { name: 'the-greatest-name-before' }
-    const now = { name: 'the-greatest-name-now' }
-    const expected = [{ action: 'changeName', ...now }]
+  it('should build the `changeName` action', () => {
+    const before = {
+      name: { en: 'en-name-before', de: 'de-name-before' },
+    }
+
+    const now = {
+      name: { en: 'en-name-now', de: 'de-name-now' },
+    }
+
+    const expected = [
+      {
+        action: 'changeName',
+        name: { en: 'en-name-now', de: 'de-name-now' },
```

Comment:
forgive the dumb question, but help me regain sanity after reading this test block 5 times because I'm missing something here.

if `now` would be

```js
{
	name: { en: 'en-name-before', de: 'de-name-now' },
}
```

What would the action look like?

Reply:
Uff, I guess not much different as the setName action always contains the whole new name to be set (as a localized string).